accounting/off_chain.go

```diff
@@ -21,6 +21,13 @@ var (
 	// lnd reflect multiple attempts to pay the same invoice.
 	errDifferentDuplicates = errors.New("duplicate payments paid to " +
 		"different sources")
+
+	// errDuplicatesNotSupported is returned when we see payments with duplicate
+	// payment hashes. This was allowed in legacy versions of lnd, but is not
+	// supported for accounting purposes. Nodes with duplicates will be required to
+	// delete the duplicates or query over a range that excludes them.
+	errDuplicatesNotSupported = errors.New("accounting not supported for " +
+		"legacy nodes with duplicate payment hashes")
```

Comment:
Because users usually don't RTFM, maybe we could add a suggestion to the error message? Like "adjust time range to skip duplicate payments" or something?